dogen/plugins/dist_git.py

```diff
@@ -114,6 +114,12 @@ def prepare(self):
                 subprocess.check_output(["git", "fetch"])
                 subprocess.check_output(["git", "checkout", "-f", self.branch])
                 subprocess.check_output(["git", "reset", "--hard", "origin/%s" % self.branch])
+                if not subprocess.call(["git", "ls-files", "--error-unmatch", "cct"],
```

Comment:
Nice trick, but I have one question.

Why do we remove `cct` directory in the `prepare` method and not in `clean` where we do it for other files?